ddtrace/contrib/langchain/patch.py

```diff
@@ -63,8 +81,7 @@ def get_version():
 )
 
 
-def _extract_model_name(instance):
-    # type: (langchain.llm.BaseLLM) -> Optional[str]
+def _extract_model_name(instance: Any) -> Optional[str]:
     """Extract model name or ID from llm instance."""
```

Comment:
Because Langchain offers support for multiple LLM providers (ex: openai, anthropic, cohere, huggingface, ...) there are probably as many unique model names as those llm providers host, but should still be bounded